codespan/src/index.rs

```diff
@@ -180,7 +180,8 @@ impl fmt::Display for ColumnOffset {
     }
 }
 
-/// A byte position in a source file
+/// A 1-indexed byte position in a source file. Zero is reserved for positions that do not
```

Comment:
It isn't really `1-indexed` either however. Since all files reside in the same "index" it is only the first file that is `1-indexed` the others start at 316, 123745, etc

Reply:
Agreed. I'll merge it if this is updated to be more accurate!